connectors/services/access_control_sync_job_execution.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+from functools import cached_property
+
+from connectors.protocol import JobStatus, JobType
+from connectors.services.job_execution import JobExecutionService
+
+DEFAULT_MAX_CONCURRENT_ACCESS_CONTROL_SYNCS = 1
```

Comment:
Should it go to default config instead?

Reply:
This is the default value when `service.max_concurrent_access_control_syncs` is not configured.

Is the `default config` somewhere we keep all the default values?

Reply:
Good point. I will address it as a follow-up PR